FakeNFT/Scenes /StatisticsPage/StatisticsPageViewController.swift

```diff
@@ -0,0 +1,117 @@
+import UIKit
+
+final class StatisticsPageViewController: UIViewController {
+    private var viewModel: StatisticsPageViewModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        view.addSubview(tableView)
+
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.topAnchor, constant: 100),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -100)
+        ])
+
+        let navigationController = UINavigationController(rootViewController: StatisticsPageViewController())
+
+        navigationController.view.translatesAutoresizingMaskIntoConstraints = false
+        let backButton = UIBarButtonItem()
+        backButton.title = ""
+        navigationItem.backBarButtonItem = backButton
+        navigationItem.rightBarButtonItem = menuButton
+
+        viewModel = StatisticsPageViewModel()
+        viewModel.onChange = updateTable
+        viewModel.getUsers()
+    }
+
+    func updateTable() {
+        tableView.reloadData()
+    }
+
+    private lazy var tableView: UITableView = {
+        let table = UITableView()
+        table.register(UserViewCell.self, forCellReuseIdentifier: "cell")
+        table.separatorInset = .init(top: 0, left: 32, bottom: 0, right: 32)
+        table.separatorColor = .asset(.grey)
+        table.delegate = self
+        table.dataSource = self
+        table.translatesAutoresizingMaskIntoConstraints = false
+        return table
+    }()
+
+    private lazy var menuButton: UIBarButtonItem = {
+        let menuButton = UIBarButtonItem(
+            image: UIImage(named: "menu"),
+            style: .plain,
+            target: self,
+            action: #selector(openMenu)
+        )
+        menuButton.tintColor = .black
```

Comment:
```suggestion
menuButton.tintColor = .asset(.black)
```

Тут тоже наверное из .asset - экстеншн у тебя есть, но не используются. Может еще где-то по коду использование цвета без .asset есть.